### PR TITLE
chore: update logs

### LINKED ISSUE #430: The verbose log for checking trust policy lacks intuitiveness
### What is not working as expected?

When using the `notation verify` command with the flag `-v`, the following logs will be shown:

> INFO Checking whether signature verification should be skipped or not
> INFO Trust policy configuration: <Trust policy was printed out here>
> INFO Check over. Trust policy is not configured to skip signature verification

The logs indicated that trust policy is being checked to see whether signature verification will be skipped or not. The last log indicated that the check is over. However, the sentence `Trust policy is not configured to skip signature verification` is not well written, there is a double negative in the sentence, which is not easy to understand.

### What did you expect to happen?

Improve the sentence `Trust policy is not configured to skip signature verification` to make it more intutive.

If trust policy is configured for signature verification, then the log can be 

> Trust policy is configured for signature verification at level strict.

If trust policy is configured to skip signature verification, then the log can be

> Trust policy is configured to skip signature verification.

### How can we reproduce it?

Use `notation verify` command with `-v` flag.

### Describe your environment

Linux OS

### What is the version of your notation-go Library?

`v1.2.0-beta.1`

### PR DESCRIPTION
**Pull request by Two-Hearts**
This PR updates logs.

Resolves #430. Also should resolve issue https://github.com/notaryproject/notation/issues/1004.

### CODE REVIEW

**Review by shizhMSFT — DISMISSED** [review]
LGTM

**Review by JeyJeyGao — DISMISSED** [review]
LGTM

**Review by JeyJeyGao — DISMISSED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by yizha1 — DISMISSED** [review]
LGTM

**Review by JeyJeyGao — DISMISSED** [review]
LGTM

File: notation.go [hunk 1]
```diff
@@ -481,10 +481,10 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, ve
 			return ocispec.Descriptor{}, nil, err
 		}
 		if skip {
-			logger.Infoln("Verification skipped for", verifyOpts.ArtifactReference)
+			logger.Infoln("Signature verification skipped for", verifyOpts.ArtifactReference)
 			return ocispec.Descriptor{}, []*VerificationOutcome{{VerificationLevel: verificationLevel}}, nil
 		}
-		logger.Info("Check over. Trust policy is not configured to skip signature verification")
```
Comment (priteshbandi) [inline notation.go:487]: IMO the existing log is more informative.

**Review by priteshbandi — COMMENTED** [review]
LGTM with a comment

File: notation.go (hunk 1 above)
Comment (Two-Hearts) [reply notation.go:487]: @yizha1 Could you share your thoughts regarding this?

Comment (Two-Hearts) [reply notation.go:487]: Updated.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**